### PR TITLE
[8.0][ADD] module website_event_registration_seat_limit

### PR DESCRIPTION
**Pull request by angelmoya**
Add module that extend event_registration_seat_limit https://github.com/OCA/event/tree/8.0/event_registration_seat_limit to limit the quantity selector on online register process

### CODE REVIEW

**Comment by tarteo** [comment]
If I change "Maximum of participants per registration" to e.g. 20 (>9), the max. number of participants in the selection box gets limited to 9.

**Comment by angelmoya** [comment]
@tarteo This feature or bug is for website_event_sale

File: website_event_registration_seat_limit/views/event_template.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<openerp>
+    <data>
+
+        <template id="event_description_full" inherit_id="website_event_sale.event_description_full" name="Event's Ticket limit">
+            <xpath expr="//select[@class='form-control']" position="replace">
+                <select t-if="ticket.seats_available or not ticket.seats_max" t-attf-name="ticket-#{ ticket.id }" class="form-control">
+                    <t t-foreach="range(ticket.event_id.registration_seats_min, min(min(ticket.event_id.registration_seats_max,ticket.seats_available or 9) + 1, 10))" t-as="nb"><option t-esc="nb"/></t>
```
Comment (tarteo) [inline website_event_registration_seat_limit/views/event_template.xml:9]: @angelmoya Isn't this line causing the problem?

Comment (angelmoya) [reply website_event_registration_seat_limit/views/event_template.xml:9]: No, problem is with clone_oca_dependencies, I don't know if problem could be with recursive dependencies...

**Comment by angelmoya** [comment]
I create an issue on mantainer_quality_tools:
https://github.com/OCA/maintainer-quality-tools/issues/349

**Comment by moylop260** [comment]
Currently the configuration of runbot is manual.
We will work on runbot_travis2docker module on next oca sprint to configure runbot automatically like as travis

**Comment by angelmoya** [comment]
Perfect @moylop260 I will be also in next oca sprint maybe I will be able to work on that

**Comment by moylop260** [comment]
Cool
I'll see you again man

**Comment by rafaelbn** [comment]
Maybe @angelmoya you can add that module in this PR in order to be able to test it if runbot don't add it through dependency

  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3153920-253-becbaf/openerp/addons/base/module/module.py", line 393, in state_update
    raise orm.except_orm(_('Error'), _("You try to install module '%s' that depends on module '%s'.\nBut the latter module is not available in your system.") % (module.name, dep.name,))
except_orm: ('Error', u"You try to install module 'website_event_registration_seat_limit' that depends on module 'event_registration_seat_limit'.\nBut the latter module is not available in your system.")

cc @Yajo @pedrobaeza  this is very interesting module

**Comment by pedrobaeza** [comment]
This module should be hosted in OCA/event, not here, as we have a "vertical" repository that is more appropriated than the general one.

**Comment by angelmoya** [comment]
Hi @pedrobaeza, there are other modules related with event and website hosted on this project:
- website_event_filter_organizer
- website_event_register_free
- website_event_register_free_with_sale

I think that it is related with the website to, or this tree modules must to changed to?

**Comment by pedrobaeza** [comment]
That was because OCA/event didn't exist in that moment, and should be moved on next version. You can also find in OCA/event another _website_event_*_ modules.

**Comment by angelmoya** [comment]
Thanks @pedrobaeza , then you can close this PR and I will do on event project

**Comment by yajo** [comment]
I believe you can close it yourself too, as the opener.

**Comment by pedrobaeza** [comment]
No problem in me closing it

**Comment by angelmoya** [comment]
New PR on https://github.com/OCA/event/pull/56

**Pull request closed without merging**